### ISSUE TITLE
Qfield 3.1.3 crash after taking a picture when the phone has already been locked once.

### ISSUE THREAD

**Issue by Amalvir** [opengisch/QField#4860]
### Describe the issue

Qfield crash when you take a picture with the Qfield camera but only after you have locked once the phone. See the reproduction steps for more details.

Maybe it is a duplicate of #4704 but the 3.1.3 with Qt 6.6.1 was supposed to fix it. Though, it seems to be a different bug because the 💥 only happen after locking and unlocking the phone, not after taking several pictures.

### Reproduction steps
1. Lauch a Qfield project with a layer with a picture field.
2. In the Qfield settings, deactivate the use of the native camera.
3. Add an entity and take a picture, validate it, everything is fine.
4. Lock your phone with its lock buttons.
5. Unlock it
6. Add a entity, add a picture in it. As soon as you will touch the big white button, qfield will 💥.

### Desktop (please complete the following information)
 - OS: Windows 10
 - QGIS Version: 3.34.1
 - QFieldSync Version: 4.6.1

### Mobile (please complete the following information)
 - Device: Samsung S20 FE 5G
 - OS: Android 13
 - QField version: 3.1.3 (14a225)

### Additional information
* Problem started happening recently, didn't happen in an older version of QField: I am not sure.
* Problem can be reliably reproduced, doesn't happen randomly: Yes
* Problem happens with all files and projects, not only some files or projects: Yes

**Comment by Amalvir** [comment]
By the way, something similar happens with the native camera. Still after locking once the phone, when you take a picture and validate it, it doesnt work and the project is reload. See the attached video for a demo.

Uploading Screen_Recording_20231216_101604_Camera.mp4…

**Comment by nirvn** [comment]
@Amalvir , I've been able to replicate this issue. It's unfortunately not a duplicate from the previous crash fixed in QField 3.1, but does seem to be another upstream issue. Looking into it some more.

**Comment by nirvn** [comment]
Sentry issue: https://opengisch.sentry.io/issues/4723524262/?project=6119013&query=is%3Aunresolved&referrer=issue-stream&statsPeriod=14d&stream_index=0

**Comment by nirvn** [comment]
Alright, I don't think there's anything we can do in QField itself. I've added a comment in the above-linked Qt issue sharing our stack trace.

Shortly after adding the comment, I stumbled on this other Qt issue which I think is essentially the same bug: https://bugreports.qt.io/browse/QTBUG-118839. We can look forward to this fix in Qt 6.6.2.

**Comment by nirvn** [comment]
Fixed in QField 3.1.5
